satpy/readers/agri_l1.py

```diff
@@ -154,7 +168,10 @@ def calibrate_to_reflectance(self, data, channel_index, ds_info):
         """Calibrate to reflectance [%]."""
         logger.debug("Calibrating to reflectances")
         # using the corresponding SCALE and OFFSET
-        cal_coef = 'CALIBRATION_COEF(SCALE+OFFSET)'
+        if self.filename_info['platform_id'] == 'FY4B':
+            cal_coef = 'Calibration/CALIBRATION_COEF(SCALE+OFFSET)'
```

Comment:
Maybe, it's better to define `CAL_ID` at the beginning like the `CHANS_ID`?

Reply:
In this case I didn't do so, as there's no confusion about what we want to load (it's always in `Calibration/` while for the other datasets it coul dbe `Data/` or `Navigation/`).